nvi-commons/src/main/java/no/sikt/nva/nvi/common/queue/DynamoDbChangeMessage.java

```diff
@@ -0,0 +1,39 @@
+package no.sikt.nva.nvi.common.queue;
+
+import static no.sikt.nva.nvi.common.utils.Validator.shouldNotBeNull;
+import static no.unit.nva.commons.json.JsonUtils.dtoObjectMapper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.UUID;
+import software.amazon.awssdk.services.dynamodb.model.OperationType;
+
+/**
+ * Simplified representation of the DynamoDB event message sent when a record is updated in the
+ * database. This is used for records that either are or are directly related to a Candidate. This
+ * is used to avoid sending the entire original DynamoDB event message, which can be too big to fit
+ * on a queue.
+ *
+ * @param candidateIdentifier the identifier of the related candidate
+ * @param entryType the type of entry being changed, which is not necessarily the same as the type
+ *     of the DAO
+ * @param operationType the type of DynamoDB operation that triggered the event
+ */
+public record DynamoDbChangeMessage(
```

Comment:
Amazing that creating a structure that actually represents the thing you're trying to manipulate with necessary accessors for the processing you intend to do makes things easier. Astounded. Have fallen off chair. I am sure you are doing something illegal, which is why I have not seen this before.